examples/ethereum-tracker/src/lib.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*!
+The Ethereum tracker for storing the values by looking at the entries in the
+Ethereum log.
```

Comment:
Maybe:

> A Linera application that reads events emitted by the `simple_token.sol` example from `linera-ethereum` and tracks the balances. This shows how Wasm contracts on Linera can consume information generated by EVM contracts on Ethereum.

(And maybe link to the Solidity contract.)

Reply:
Ok. But the solidity contract is exactly that `simple_token.sol` so I am not sure what we can add as more information.

Reply:
Yes, maybe a link isn't needed if we mention that it's `simple_token.sol` in `linera-ethereum`.